admin/tabs/activity/clienterrors/ClientErrorModel.js

```diff
@@ -49,6 +49,12 @@ export class ClientErrorModel {
         ]
     });
 
+    constructor() {
```

Comment:
Would it be more straightforward potentially to just have exportFilename potentially be a closure as well as a string?

Would be nice to avoid the complexity of an autorun for this pretty common use case